### PR TITLE
Build with stack

### PR DESCRIPTION
**Pull request by diogob**

### CODE REVIEW

**Comment by PierreR** [comment]
Well now you forbid anyone to build it with `aeson-0.9` (which is the default on arch-linux) ? Why not `aeson >= 0.8` ?

**Comment by diogob** [comment]
@PierreR the syntax was generated by the stack, I just filled in the extra-deps array. The only problem with aeson >= 0.8 is that we won't be able to build using stack right now, which is not a bad thing in itself. I was actually looking for the feedback you've just given about arch ;)

**Comment by diogob** [comment]
Oh, sure, I'm changing the array syntax

**Comment by diogob** [comment]
@PierreR my mistake, we can build with stack as long as we make a small change to one of the specs so it can pass with any version of aeson >= 0.8. Now it should be fine.

File: test/Feature/InsertSpec.hs [hunk 1]
```diff
@@ -87,12 +87,7 @@ spec = afterAll_ resetDb $ around withApp $ do
 
     context "with invalid json payload" $
       it "fails with 400 and error" $
-        post "/simple_pk" "}{ x = 2"
-          `shouldRespondWith` ResponseMatcher {
-            matchBody    = Just [json| {"message":"Failed to parse JSON payload. Failed reading: not a valid json value"} |]
-          , matchStatus  = 400
-          , matchHeaders = []
-          }
+        post "/simple_pk" "}{ x = 2" `shouldRespondWith` 400
```
Comment (begriffs) [inline test/Feature/InsertSpec.hs]: Sure, checking for the error status alone should be fine.

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,4 @@ cabal.sandbox.config
 hscope.out
 codex.tags
 .anvil
+.stack-work
```
Comment (begriffs) [inline .gitignore]: Is `.stack-work` a scratch directory created by Stack?

Comment (diogob) [inline .gitignore]: Yeah, it install all the generated binaries, logs and local packages (the ones that are not included in stackage snapshot). The plus side is that one does not need a sandbox to develop with stack :D

**Comment by PierreR** [comment]
:+1:  @diogob thanks for the PR. Stack looks great !

**Pull request merged**